libs/atqr/domain/src/lib/challenge/challenge.entity.ts

```diff
@@ -102,4 +102,7 @@ export class Challenge {
       return false;
     }
   }
```

Comment:
Are there any business rules that should be checked before updating status?

Reply:
maybe check if the status is overdue?

Reply:
Sorry for taking so long to answer, actually I don't know the answer either, Maybe @svallory has something to say. But in the meantime I think this branch in ready to be merged and if the need arises we implement more business rules later